Graphics/GraphicsEngine/interface/GraphicsTypes.h

```diff
@@ -3485,6 +3485,12 @@ struct EngineGLCreateInfo DILIGENT_DERIVE(EngineCreateInfo)
     /// Use IRenderDevice::GetDeviceInfo().NDC to get current NDC.
     Bool         ZeroToOneNDZ DEFAULT_INITIALIZER(false);
 
+    /// The GPU preference allows you to request either the integrated or dedicated GPU
+    /// on systems having both onboard and dedicated GPUs. Currently this works only on Windows and Linux
+    /// - On Windows this is done by setting the `NvOptimusEnablement` and  `AmdPowerXpressRequestHighPerformance` 
+    /// - On Linux this affects the `DRI_PRIME` environment variable that's used by Mesa drivers that support PRIME.
+    ADAPTER_TYPE PreferredAdapterType DEFAULT_INITIALIZER(ADAPTER_TYPE_UNKNOWN);
```

Comment:
It is also important to add here that the executable target needs to be linked with the `Diligent-GLAdapterSelector`